src/main/steps/applicant1/hub-page/sole/latest-update-content/final-order-complete.njk

```diff
@@ -0,0 +1,9 @@
+{% if isFinalOrderCompleteState %}
```

Comment:
this is the exact same template as the one above, so we can just reference the other one